sidebars.js

```diff
@@ -47,6 +47,7 @@ const sidebars = {
     'quickstart/README',
     'quickstart/create-a-local-test-network',
     'quickstart/fund-a-local-test-network',
+    'quickstart/avalanche-faucet',
```

Comment:
Debating if this should be here or under Subnets. If here, it should be below `Fuji Workflow`.

Reply:
Let's move it to the Subnets section after title change.